src/partials/movie-modal.html

```diff
@@ -1,2 +1,72 @@
 <!-- adding and removing class "show" -->
-<div id="movie-lightbox"></div>
+<div class="backdrop-movie">
+  <div id="movie-lightbox">
```

Comment:
Ten movie lightbox możesz zmienić nazwę bo ostatecznie podjołem decyzje że to nie jest lightbox a modal

I prosił bym abyś robiła nazwy klas konwencją BEM ale to juz szczegóły

Reply:
done